web-api/jest-unit.config.ts

```diff
@@ -28,6 +27,12 @@ const config: Config = {
     '!src/persistence/sqs/getMessages.ts',
     '!src/persistence/messages/*.ts',
     '!src/persistence/dynamo/**/*.ts',
+    '!src/lambdas/websockets/websockets.ts',
+    '!src/lambdas/websockets/switch-colors-cron.ts',
+    '!src/lambdas/cognitoAuthorizer/worker-handler.ts',
+    '!src/lambdas/api/api.ts',
+    '!src/lambdas/api-public/api-public.ts',
+    '!src/lambdas/migration/utilities/getRecordSize.ts',
```

Comment:
I would categorize both`websockets.ts` and `api-public.ts` as not testable as they are just skeletons for other functions that are defined in other files or call a 3rd party service.

In my opinion, `switch-colors-cron.ts`, `worker-handler.ts`, `api.ts` are not worth testing (and weren't tested before this PR) because most of them are covered less explicitly through e2e tests and the unit tests for the files may be too abstract to be useful if they fail. Although I would be happy to add coverage if you think it would be useful!

For `getRecordSize.ts`, I remember Mike and I tried to add coverage for that for a day while we were adding that helper function and ultimately decided together that the juice wasn't worth the squeeze. Again, I'd be happy to add coverage if you disagree 😄